cmd/kube-proxy/app/server_test.go

```diff
@@ -386,6 +386,10 @@ func TestProcessHostnameOverrideFlag(t *testing.T) {
 // TestOptionsComplete checks that command line flags are combined with a
 // config properly.
 func TestOptionsComplete(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Skipping test on Windows.")
```

Comment:
question: why do we skip these tests? Can this never be run on Windows? If there's a good reason, it should be added as comment. Same for skips below.